toolkits/nature/packages/nature-facet/scss/50-components/facet.scss

```diff
@@ -0,0 +1,171 @@
+.c-facet {
+	background-color: #f3f3f3;
```

Comment:
Should we create a variable to allow consumers to change background colour relevant to their brand? If this is a grey, should we use the greyscale function in toolkits?